v23.1/architecture/transaction-layer.md

```diff
@@ -117,16 +117,16 @@ Note that closed timestamps are valid even if the leaseholder changes, since the
+The timestamp for any transaction, especially a long-running transaction, could be [pushed](#timestamp-cache). An example is when a transaction encounters a key written at a higher timestamp. When this kind of [contention](../performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention) happens between transactions, the closed timestamp **may** provide some mitigation. Increasing the closed timestamp interval may reduce the likelihood that a long-running transaction's timestamp is pushed and must be retried. Thoroughly test any adjustment to the closed timestamp interval before deploying the change in production, because such an adjustment can have an impact on:
 
-- [Follower reads](../follower-reads.html): Reads will be forced to read from the leaseholder, which may affect latency/throughput.
-- [Change data capture](../change-data-capture-overview.html): There could be a lag in changefeed messages.
-- [Statistics collection](../cost-based-optimizer.html#table-statistics): Collection can place more load on the leaseholder.
+- [Follower reads](../follower-reads.html): Latency or throughput may be increased, because reads are served only by the leaseholder.
+- [Change data capture](../change-data-capture-overview.html): Latency of changefeed messages may be increased.
+- [Statistics collection](../cost-based-optimizer.html#table-statistics): Load placed on the leaseholder may increase during collection.
 
-While increasing the closed timestamp may decrease retryable errors, it may also increase lock latencies. For example, a long-running transaction (txn 1) holds [write locks](#writing) on keys at time t=1, and txn 2 is waiting to [read](#reading) those same keys at t=1. The following scenarios may occur depending on whether the closed timestamp has been increased or not:
+While increasing the closed timestamp may decrease retryable errors, it may also increase lock latencies. Consider an example where the long-running transaction `txn 1` holds [write locks](#writing) on keys at time `t=1`, and the long-running transaction `txn 2` is waiting to [read](#reading) those same keys at `t=1`. The following scenarios may occur depending on whether or not the closed timestamp has been increased:
```

Comment:
```suggestion
While increasing the closed timestamp may decrease retryable errors, it may also increase lock latencies. Consider an example where:

1. The long-running transaction `txn 1` holds [write locks](#writing) on keys at time `t=1`.
1. The long-running transaction `txn 2` is waiting to [read](#reading) those same keys at `t=1`.

The following scenarios may occur depending on whether or not the closed timestamp has been increased:
```
This is complicated and it might reduce the cognitive load to break it up just a little. WDYT?